### ISSUE TITLE
log files of tools are stored repeatedly

### ISSUE THREAD

**Issue by crhf** [nus-apr/cerberus#47]
for example,
```
$ tree output/logs
output/logs
├── C5-arja-defects4j-Chart-1-c2e0be1d
│   ├── C5-arja-1-output.log
│   └── logs
│       ├── C5-arja-1-output.log
│       └── logs
│           └── C5-arja-1-output.log
├── C5-cardumen-defects4j-Chart-1-a08ad419
│   ├── C5-cardumen-1-output.log
│   └── logs
│       ├── C5-cardumen-1-output.log
│       └── logs
│           └── C5-cardumen-1-output.log
...
```

C5-arja-1-output.log and logs/C5-arja-1-output.log are the same, while logs/logs/C5-arja-1-output.log has fewer lines. Ditto cardumen.

**Comment by rshariffdeen** [comment]
Thanks for creating the issue, I will look into it. 
In the meantime if you have a fix, would be great if you can submit a PR

**Comment by Marti2203** [comment]
Fixed in commit b2bc16f4
